modules/dcache/src/main/java/diskCacheV111/services/PoolStatisticsV0.java

```diff
@@ -148,87 +148,87 @@
 
     private static final String DEFAULT_AUTHOR = "&copy; dCache.org ";
 
-    private static final ThreadLocal<SimpleDateFormat> _pathFromDate =
-            new ThreadLocal<SimpleDateFormat>()
+    private static final ThreadLocal<DateTimeFormatter> _pathFromDate =
```

Comment:
as DateTimeFormatter is thread-safe you don't need to to ThreadLocal anymore.

Reply:
All ThreadLocal-variables have been removed and changed